File: examples/thumbnail/screenshot/node/takeScreenshotOfAUrl.js
const { ThumbnailService } = require("micro-js-client/thumbnail");

// Create a thumbnail screenshot by passing in a url, height and width
async function takeScreenshotOfAurl() {
  let thumbnailService = new ThumbnailService(process.env.MICRO_API_TOKEN);
  let rsp = await thumbnailService.screenshot({
    height: 600,
    url: "https://google.com",
    width: 600,
  });
  console.log(rsp);
}

takeScreenshotOfAurl();

File: examples/thumbnail/screenshot/go/takeScreenshotOfAUrl.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/thumbnail"
)

// Create a thumbnail screenshot by passing in a url, height and width
func TakeScreenshotOfAurl() {
	thumbnailService := thumbnail.NewThumbnailService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := thumbnailService.Screenshot(&thumbnail.ScreenshotRequest{
		Height: 600,
		Url:    "https://google.com",
		Width:  600,
	})
	fmt.Println(rsp, err)
}
